DockerFIle

```diff
@@ -5,4 +5,5 @@ RUN npm install
 RUN npm run build
 WORKDIR /
 RUN mkdir /FEProdBuild
-COPY /frontend/build /FEProdBuild
\ No newline at end of file
+COPY /frontend/build /FEProdBuild
+WORKDIR /
```

Comment:
Is the name of the file correct?  DockerFIle with a capital I?

Reply:
no the correct spelling is `Dockerfile` but it looks like it didnt work from looking at the digital ocean logs. Ill have to try again or do more research.